### ISSUE TITLE
Dashboad App shows (mostly) empty screen. Only logo, the red notification icon and the avatar is shown.

### ISSUE THREAD

**Issue by lars-becker** [nextcloud/server#22966]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### problem description  / Steps to reproduce

After installing NC 20 Beta 4 most things work as usual, but the new dashboard app only shows a mostly empty screen. Only the logo, the red notification icon and the user avatar is shown.

The debugging console of Firefox shows:

```
01:52:33.095 Content Security Policy: The page’s settings blocked the loading of a resource at data: (“media-src”).
01:52:33.205 Content Security Policy: The page’s settings blocked the loading of a resource at inline (“script-src”). 2 content-script.js:40:65
```

These messages are followed by a lot of TypeErrors:

```
01:52:34.932 Uncaught TypeError: OCA.Files.DetailTabView is undefined
    <anonymous> https:///apps/activity/js/activitytabview.js:29
    <anonymous> https:///apps/activity/js/activitytabview.js:229
activitytabview.js:29:24
[...]
01:52:34.935 Uncaught TypeError: n.Files.DetailTabView is undefined
    <anonymous> commentstabview.js:21
    <anonymous> commentstabview.js:756
    Webpack 3
        n
        <anonymous>
        <anonymous>
commentstabview.js:21:23
[...]
01:52:34.938 Uncaught TypeError: OCA.Files.DetailTabView is undefined
    <anonymous> versionstabview.js:18
    Webpack 3
        n
        <anonymous>
        <anonymous>
versionstabview.js:18:7
[...]
01:52:35.523 No OCA.Files app found, viewer is now in standalone mode Viewer.vue:295
[...]
01:52:37.281 TypeError: window.OCA.Accessibility is undefined
    data App.vue:144
    VueJS 7
    <anonymous> main.js:21
    Webpack 3
vue.runtime.esm.js:1888:12
```

and so on.  In  the Apache configuration there's no CSP defined which could interfere with Nextclouds settings.

### Expected behaviour
Show the dashboard.

### Server configuration

**Operating system:** Ubuntu 18.04 LTS

**Web server:** Apache2 (2.4.46-2+ubuntu18.04.1+deb.sury.org+1) with php-fpm

**Database:** MySQL  (5.7.31-0ubuntu0.18.04.1)

**PHP version:** php-fpm (7.4.9-1+ubuntu18.04.1+deb.sury.org+1)

**Nextcloud version:** 20 Beta 4

**Updated from an older Nextcloud/ownCloud or fresh install:** Update from older version since ownCloud 5.
### Client configuration
**Browser:** Firefox 80.0.1 (can be reproduced with Google Chrome too)

**Operating system:** Archlinux

### Logs
No useful entries in the server or nextcloud logs.

**Comment by lars-becker** [comment]
The problem was a `custom_csp_policy`  directive in `config.php`.

**Comment by lars-becker** [comment]
Well that solved the error with the Content Security Policy, but the screens's still empty even after clearing all server side and client side caches and lots of type errors prevail.  Any advice on how to debug this?  Sadly I'm not much of an frontend/JS developer.

**Comment by axel-rank** [comment]
Yes, same in Nextcloud 20rc1.
Dashboard shows empty screen, only avatar is shown.

**Comment by lars-becker** [comment]
Stills persists with NC20.

**Comment by lars-becker** [comment]
closing in favor of #23163

**Comment by thangisme** [comment]
Don't know if this's related. But I just installed Nextcloud. When I was visiting login page, there's no login form. I opened console and found that some script were being blocked by CSP
